src/rpc_api/data_types.rs

```diff
@@ -399,6 +399,19 @@ impl HasLotusJson for PendingBeneficiaryChange {
         }
     }
 }
+#[derive(Debug, Clone, Serialize, Deserialize, Eq, PartialEq)]
+#[serde(rename_all = "PascalCase")]
+pub struct ApiReceipt {
```

Comment:
Not sure if it's the practice all of you do with RPC methods, but it'd be great to document the fields we return.